server/database/schema.sql

```diff
@@ -0,0 +1,21 @@
+CREATE DATABASE Ingredients;
+
+USE Ingredients;
+
+CREATE TABLE Users (
+  ID INT NOT NULL AUTO_INCREMENT,
+  Name varchar (50) NOT NULL,
+  Username varchar (50), NOT NULL,
+  Passwords varchar(50) NOT NULL,
+  Email varchar (50) NOT NULL,
+  PRIMARY KEY (ID)
+);
+
+CREATE TABLE Foods (
```

Comment:
This table is on a separate ticket. I would review that ticket and ensure that it matches the guidelines.